apps/ledger-live-desktop/src/renderer/families/evm/StakeFlowModal/index.tsx

```diff
@@ -1,23 +1,172 @@
-import React from "react";
-import { Account } from "@ledgerhq/types-live";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/currencies/index";
```

Comment:
I think you can divide this component in sub components to avoid a large file like this one. For example you can create a file for all your styled-components, Header.tsx for your header part, etc ...